packages/syntax-core/src/Button/constants/color.ts

```diff
@@ -0,0 +1,66 @@
+export function classicColor(
+  color:
+    | "primary"
+    | "secondary"
+    | "tertiary"
+    | "quaternary"
+    | "destructive-primary"
+    | "destructive-secondary"
+    | "destructive-tertiary"
+    | "branded"
+    | "success"
+    | "success-primary"
+    | "success-secondary"
+    | "inverse",
+):
+  | "primary"
+  | "secondary"
+  | "tertiary"
+  | "destructive-primary"
+  | "destructive-secondary"
+  | "branded"
+  | "success"
+  | "inverse" {
+  if (color === "quaternary") {
+    return "inverse";
+  } else if (color === "destructive-tertiary") {
+    return "destructive-secondary";
+  } else if (color === "success-primary" || color === "success-secondary") {
+    return "success";
+  }
+  return color;
```

Comment:
Is this here so the classic buttons don't break if we convert them to cambio? Is it going to be removed eventually?

Reply:
@btorndorff that is correct - added so both themes work with each other. Goal is to remove the `classic` them eventually